### DISCUSSION TITLE
Amethyst 0.16.0

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1297] [Announcements]
* Amethyst now more robustly supports configuration files. See documentation: [Configuration Files](https://github.com/ianyh/Amethyst/blob/development/docs/configuration-files.md) #1251
* Amethyst now supports a beta of custom layouts. See documentation: [Custom Layouts](https://github.com/ianyh/Amethyst/blob/development/docs/custom-layouts.md) #1134, #51, #1126 
* Fix the presence of ghost windows in the bsp layout. Thanks, @LudwigWS! #1249, #1246
* Fix the layout HUD when multiple screen are arranged vertically. Thanks, @Henkru! #1162, #1163
* Make TwoPane layout adaptive to screen orientation. Thanks, @mwz! #1182
* Improve main window swapping with TwoPane layout. Thanks, @rickythefox! #1241, #1242, #1234 
* Add 4Column layout for ultra-wide screens. Thanks, @reyk! #1154 
* Throw window to any space across multiple screens. Thanks, @dozzman! #1209 
* Fix an issue that caused Amethyst to fail to tile entirely. #1250 

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.16.0'>Amethyst 0.16.0</a>.</em>

**Comment by TomEstelrich** [comment]
Are there any plans to release 0.16.1 with the latest bug fixes for Ventura soon? As someone who uses Amethyst on my daily basis I'd love to see those bugs fixed. 👍
